tests/test_stream.py

```diff
@@ -3,94 +3,89 @@
 import pytest
 
 from apepay import exceptions as apepay_exc
-from apepay.utils import TODO_UNDO
 
 
-def test_todo_undo():
```

Comment:
during some debugging, this test was never meant to be merged, but it did anyway, so just undoing that nonsense.